spec/factories/application_form.rb

```diff
@@ -58,13 +58,17 @@
 
     trait :with_equality_and_diversity_data do
       equality_and_diversity do
-        ethnicity = Class.new.extend(EthnicBackgroundHelper).all_combinations.sample
+        all_ethnicities = Class.new.extend(EthnicBackgroundHelper).all_combinations
+        if RecruitmentCycle.current_year < 2023
```

Comment:
might be good to have something like `Hesa::CODE_CHANGES_2023`

Reply:
I can do it but this is only used in specs to create certain data depending of the year. Should I still?

Reply:
i wouldn't bother if it were only used in this test, but i think it'd be useful in the disabilities PR also.